config.yml.erb

```diff
@@ -249,6 +249,8 @@ aws_secret_key:
 
 # Amazon Future Engineer integration
 afe_pardot_form_handler_url:
+csta_jotform_api_key: !Secret
+csta_jotform_form_id: !Secret
```

Comment:
Note: CSTA has provided us with a test form and a live form.  I have all environments pointing at the test form for now.